### DISCUSSION TITLE
APSIM Data Assimilation Using Jupyter Notebook

### DISCUSSION THREAD

**Discussion by pakagronomist** [APSIMInitiative/ApsimX#9428] [General]
### Describe the new feature

Respected All, 
As you know these days, growth models application is in trend with data assimilation to improve models accuracy. Many articles available for WOFOST or APSIM use for data assimilation through Ensemble kalman filters. I am  in need of  guidance how it can be done that I use Ensemble Kalman filter for data assimilation and then these assimilated values of LAI, Soil moisture may pass through APSIM model to predict crop yield instead of using own simulated values of LAI or Soil moisture using Jupyter Notebook. I have run APSIM and Ensemble Kalman filters in Jupyter notebook and now I am struck by how filter assimilated values integrate with APSIM Model (in Jupyter Notebook)  to predict yield based on assimilated data. Can you please guide me about this execution? I am not much versed with languages and models modification being an agriculturist. I will be very thankful for your guidance and support.

**Comment by pakagronomist** [comment]
Anyone can help me out what will be the  code that EnK filter assimilated (mainly LAI) data should be used by APSIM NG to predict yield instead of APSIM simulated LAI in Juypternotebook. I will be thankful. Regards.

**Comment by BrianCollinss** [comment]
I'm not sure if your question is quite clear. Besides, this is a very specific and advanced use of the model and any help would require substantial information about your objectives and existing setup.

**Reply by BrianCollinss** [reply]
@pakagronomist Is this a PhD or externally funded project?

**Reply by pakagronomist** [reply]
No Sir, its not a funded project. Only study for my thesis work completion. @BrianCollinss I am grateful to you for your always guidance and cooperation in personal contact as well.

**Reply by pakagronomist** [reply]
Sir objective is APSIM yield estimation improvement by assimilating remote sensing data observations

**Reply by BrianCollinss** [reply]
@pakagronomist What university are you working at?

**Reply by pakagronomist** [reply]
Beihang University, Beijing China

**Reply by BrianCollinss** [reply]
> Sir objective is APSIM yield estimation improvement by assimilating remote sensing data observations

This is something that our team is involved in as well. You may benefit from having a supervisor with experiences directly related to your project. Contact me if you like to discuss this.

**Reply by pakagronomist** [reply]
Surely Sir, it will a pleasure for me.
